modules/light-clients/08-wasm/types/export_test.go

```diff
@@ -6,11 +6,3 @@ package types
 
 // MaxWasmSize is the maximum size of a wasm code in bytes.
 const MaxWasmSize = maxWasmSize
```

Comment:
Unrelated to this PR, but do you think its possible to remove this? I think we have a public function which returns `maxWasmSize` IIRC

Reply:
ah we dont (at least couldn't find it). Though I cant see the benefit in making it public its a `const` so no worries of someone rugging us by changing this value, we could just export it too